### PR TITLE
refactor(blog): pass meta to source.title for better customization

### PR DESCRIPTION
**Pull request by vinicius73**
This enables greater path customization power.

### CODE REVIEW

File: packages/blog/src/blueprint/source.js [hunk 1]
```diff
@@ -45,7 +45,11 @@ const source = {
 
   // path() determines the final URL path of a Markdown source
   // In `blog` mode, the default format is /YYYY/MM/DD/<slug>
-  path (fileName, { title, published }) {
+  path (fileName, { title, published, meta = {} }) {
```
Comment (pimlie) [inline packages/blog/src/blueprint/source.js]: You could destructure the slug already in the argument, no need for using the meta arg here

**Comment by pimlie** [comment]
Thanks, its ok with me to change this.

I would like to ask you to please use conventional pr and commit titles, we use those to create the changelog.

File: packages/blog/src/blueprint/source.js (hunk 1 above)
Comment (vinicius73) [reply packages/blog/src/blueprint/source.js]: I choose this way because another option would conflict with `const slug` below

Comment (pimlie) [reply packages/blog/src/blueprint/source.js]: ~~const~~ ;)

But yes, I overlooked that indeed! So ok to leave it I guess

**Pull request merged**